code/game/machinery/modular_shield.dm

```diff
@@ -388,6 +388,12 @@
 	icon_state = "recharge_port"
 	density = TRUE
 
+	///This var is what determines if performance modules can connect to this module
```

Comment:
what is a performance module

Reply:
performance / boost whatever , its just anything thats not a node i should probably rewrite it.

Reply:
rewritten